integration/testrunner/db.go

```diff
@@ -24,42 +24,35 @@ import (
 )
 
 // queryIfAuditLogExists queries the DB and checks if audit log contained in the query exists or not.
-func queryIfAuditLogExists(ctx context.Context, tb testing.TB, query *bigquery.Query, expectedNum int64) (bool, error) {
+func queryIfAuditLogExists(ctx context.Context, tb testing.TB, query *bigquery.Query) ([]bigquery.Value, error) {
 	tb.Helper()
 
 	job, err := query.Run(ctx)
 	if err != nil {
-		return false, fmt.Errorf("failed to run query: %w", err)
+		return nil, fmt.Errorf("failed to run query: %w", err)
 	}
 
 	if status, err := job.Wait(ctx); err != nil {
-		return false, fmt.Errorf("failed to wait for query: %w", err)
+		return nil, fmt.Errorf("failed to wait for query: %w", err)
 	} else if err = status.Err(); err != nil {
-		return false, fmt.Errorf("query failed: %w", err)
+		return nil, fmt.Errorf("query failed: %w", err)
 	}
-
 	it, err := job.Read(ctx)
 	if err != nil {
-		return false, fmt.Errorf("failed to read job: %w", err)
+		return nil, fmt.Errorf("failed to read job: %w", err)
 	}
-
 	var row []bigquery.Value
 	if err = it.Next(&row); err != nil {
-		return false, fmt.Errorf("failed to get next row: %w", err)
-	}
-
-	// Check if the matching row count is equal to expected, if yes, then the audit log exists.
-	tb.Logf("Found %d matching rows", row[0])
-	result, ok := row[0].(int64)
-	if !ok {
-		return false, fmt.Errorf("error converting query results to integer value (got %T)", row[0])
+		return nil, fmt.Errorf("failed to get next row: %w", err)
 	}
-	return result >= expectedNum, nil
+	return row, nil
 }
 
 func makeQuery(bqClient bigquery.Client, id, queryString string) *bigquery.Query {
 	bqQuery := bqClient.Query(queryString)
-	bqQuery.Parameters = []bigquery.QueryParameter{{Value: id}}
+	if id == "" {
+		fmt.Print("hahaha")
+	}
```

Comment:
?

Reply:
Sorry, this was an oversight, I just removed this.